dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/util/DesignTemplateUtil.java

```diff
@@ -358,17 +361,24 @@ private static void setLayoutBody ( TemplateLayout layout, Document templateDraw
      * @param splitBody
      * @return
      */
-    private static List<String> getColumnContainers ( Element splitBody ) {
+    private static List<ContainerUUID> getColumnContainers (Element splitBody ) {
 
         //parseContainer regex
         Pattern parseContainerPatter = Pattern.compile( "(?<=#parseContainer\\(').*?(?='\\))" );
```

Comment:
Since the Pattern is a thread-safe object you can create an static final of it as part of the class